### ISSUE TITLE
Local Newline behavior isn't consistent with the docs

### ISSUE THREAD

**Issue by max-sixty** [hairyhenderson/gomplate#901]
I imagine this is something I'm doing incorrectly. Running some of the examples locally, I don't see the same result as the docs. Any idea what the cause could be?

Here are a couple of examples:

https://docs.gomplate.ca/functions/strings/#examples-7
```
$ gomplate -i '{{ range ("foo:bar:baz" | strings.SplitN ":" 2) }}{{.}}{{end}}'
foobar:baz

# docs:
foo
bar:baz
```

https://docs.gomplate.ca/functions/strings/#examples-6
```
$ gomplate -i '{{range ("Bart,Lisa,Maggie" | strings.Split ",") }}Hello, {{.}} {{end}}'
Hello, Bart Hello, Lisa Hello, Maggie

# docs:
Hello, Bart
Hello, Lisa
Hello, Maggie
```

I tried on both bash and fish. I'm using `gomplate version 3.7.0` on MacOS. 

In lieu of the newlines not being added, can I add them myself? Adding `\n` causes those characters rather than a newline to appear.

Thanks!

**Comment by hairyhenderson** [comment]
Hi @max-sixty - thanks for logging this!

Your behaviour is correct, the docs are wrong. To get the behaviour you want, you are indeed correct, you need to add a `\n`. There's a few ways to do this that I can think of off the top of my head:

1. you could simply add a newline character inside the template block (I think this is what I intended in the docs):

```console
$ gomplate -i '{{ range ("foo:bar:baz" | strings.SplitN ":" 2) }}{{.}}
{{end}}'
foo
bar:baz
```

2. you could output a `\n` character in a template directive:
```console
$ gomplate -i '{{ range ("foo:bar:baz" | strings.SplitN ":" 2) }}{{.}}{{ "\n" }}{{end}}'
foo
bar:baz
```

3. you could use `print` to print both the element and the `\n` in the same command:
```console
$ gomplate -i '{{ range ("foo:bar:baz" | strings.SplitN ":" 2) }}{{ print . "\n" }}{{end}}'
foo
bar:baz
```

Either way, thanks again, this is a docs bug! 😅

**Comment by hairyhenderson** [comment]
Fixed in #906! The docs site should be up-to-date within the next couple of minutes.

**Comment by max-sixty** [comment]
Great, thanks @hairyhenderson !
